### PR TITLE
fix: convert Path to str. update notebook

### LINKED ISSUE #113: PyWPS does not accept Path objects
## Description

Some outputs are passed Path objects, which are not supported by pywps 4.2.4.

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #113 

Changes:

* Fixed wps_xclim_indices. Possibly same issue elsewhere, but wondering why it didn't pop up before. 
* Updated notebook.  finch notebook in pavics-sdi can be deleted.

### CODE REVIEW

File: docs/source/notebooks/finch-usage.ipynb [hunk 1]
```diff
@@ -81,9 +101,9 @@
      "text": [
       "ProcessSucceeded\n",
       "frost_daysResponse(\n",
-      "    output_netcdf='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
-      "    output_log='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/log.txt',\n",
-      "    ref='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/input.meta4'\n",
+      "    output_netcdf='http://localhost:5000/outputs/13b909e4-686e-11ea-a94b-b052162515fb/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
```
Comment (tlvu) [inline docs/source/notebooks/finch-usage.ipynb]: Revert all output url from `localhost` to `pavics.ouranos.ca`?  With the new regex, leaving it as `localhost` will pass Jenkins but given this notebooks is used as tutorial, it's weird the output url do not match the host.

Your call, for me what's most important is Jenkins is not broken.

File: docs/source/notebooks/finch-usage.ipynb [hunk 2]
```diff
@@ -98,155 +118,34 @@
...
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Metalink content-type detected.\n",
+      "Downloading to /tmp/tmp8fi43lm1/frost-days_SRES-A2-experiment_20460101-20650101.nc.\n"
```
Comment (tlvu) [inline docs/source/notebooks/finch-usage.ipynb]: This random path most likely will break Jenkins.  But I don't have an updated Finch server to test.  Will do a blind `output-sanitize.cfg` fix.

**Comment by tlvu** [comment]
Travis-CI error that do not look like regex related.  Did you "Kernel -> Restart kernel and run all cells"?
```
_______________ docs/source/notebooks/finch-usage.ipynb::Cell 4 ________________

Notebook cell execution failed

Cell 4: Cell outputs differ

Input:

ds, log, metalink = resp.get(asobj=True)

Traceback:

Missing output fields from running code: {'stdout'}

_______________ docs/source/notebooks/finch-usage.ipynb::Cell 5 ________________

Notebook cell execution failed

Cell 5: Cell outputs differ

Input:

ds

Traceback:

Unexpected output fields from running code: {'text/html'}
```

File: docs/source/notebooks/finch-usage.ipynb (hunk 1 above)
Comment (huard) [reply docs/source/notebooks/finch-usage.ipynb]: But how do I update the notebook before the working server is live at pavics.ouranos.ca ?

**Comment by huard** [comment]
I thought I did... apparently not.

**Comment by tlvu** [comment]
@huard I just push new change to make the notebook backward compatible with old server, you'll have to pull and re-run all cells again.

**Comment by tlvu** [comment]
> I thought I did... apparently not.

Oh it might be that your jupyterlab and notebook (update both just in case) is old and the output is recorded differently from the latest version from Travis-CI which cause the diff.

File: docs/source/notebooks/finch-usage.ipynb (hunk 1 above)
Comment (tlvu) [reply docs/source/notebooks/finch-usage.ipynb]: > But how do I update the notebook before the working server is live at pavics.ouranos.ca ?

Manual, just for that cell ;(  Yeah it's ugly.  You can leave it as `localhost` as well, won't break Jenkins.  Users running the tutorials notebook might not notice as well.

**Pull request merged**